google/cloud/dialogflow_cx/quickstart/quickstart.cc

```diff
@@ -13,22 +13,30 @@
 // limitations under the License.
 
 #include "google/cloud/dialogflow_cx/agents_client.h"
-#include "google/cloud/project.h"
+#include "google/cloud/common_options.h"
 #include <iostream>
 #include <stdexcept>
 
 int main(int argc, char* argv[]) try {
-  if (argc != 2) {
-    std::cerr << "Usage: " << argv[0] << " project-id\n";
+  if (argc != 3) {
+    std::cerr << "Usage: " << argv[0] << " project-id region-id\n";
     return 1;
   }
 
+  auto const project = std::string{argv[1]};
+  auto const region = std::string{argv[2]};
```

Comment:
Just an observation. (Assuming REST and gRPC endpoints are the same)...

`dialogflow.googleapis.com` is listed in the [CX v3 endpoints list](https://cloud.google.com/dialogflow/cx/docs/reference/rest/v3-overview#rest_endpoints). Is it possible to use it?

Looking ahead, it seems like `dialogflow.googleapis.com` is the only reasonable choice for our testing project in the [ES v2 endpoints list](https://cloud.google.com/dialogflow/es/docs/reference/rest/v2-overview#rest_endpoints)

Reply:
Turns out I can use the `dialogflow.googleapis.com` but I do need to change the authority.  The request is rejected otherwise.